src/main/java/com/google/api/codegen/transformer/java/JavaModelTypeNameConverter.java

```diff
@@ -91,6 +91,11 @@ public JavaModelTypeNameConverter(String implicitPackageName) {
   }
 
   @Override
+  public TypeName getTypeName(String shortName) {
```

Comment:
I think the shortness of this name is misleading as to what it does. By default, any methods in this set of classes that take a String treat it as a full name.